src/expiring_hash_map.rs

```diff
@@ -172,7 +172,7 @@ where
     ///                 println!("Expired: {}", val);
     ///                 break;
     ///             }
-    ///             Some(Err(error)) => panic!(format!("Timer error: {:?}", error)),
+    ///             Some(Err(error)) => panic!("Timer error: {:?}", error),
```

Comment:
Is this related?

Reply:
It's the sole warning that came up in a run reported as a failure.